### PR TITLE
[ci:component:github.com/gardener/machine-controller-manager-provider-aws:v0.4.1->v0.5.0]

### PR DESCRIPTION
**Pull request by gardener-robot-ci-1**
**Release Notes*:
``` improvement user github.com/gardener/machine-controller-manager-provider-aws #32 @himanshu-kun
Allow setting of `placement` parameters on AWS EC2 instances using a specific key on machine annotations. For setting the annotation refer to the doc provided [here](https://github.com/gardener/machine-controller-manager-provider-aws/blob/262597d71db8dfbc472f189dec0cb54276ae0656/kubernetes/machine.yaml#L20).
```

``` improvement operator github.com/gardener/machine-controller-manager-provider-aws #29 @shin-nien
Support for targeting an AWS Resource Group containing capacity reservations. Refer AWS documentation [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/capacity-reservations-using.html#create-cr-group).
```

``` improvement operator github.com/gardener/machine-controller-manager-provider-aws #28 @prashanth26
Improved documentation to add sample deployment files.
```

``` improvement operator github.com/gardener/machine-controller-manager-provider-aws #26 @prashanth26
Allow deletion of machine whose providerID is empty.
```

``` other operator github.com/gardener/machine-controller-manager-provider-aws #23 @ialidzhikov
Check for misconfigured PodDisruptionBudgets when Pod eviction fails during Node drain.
```

``` breaking developer github.com/gardener/machine-controller-manager-provider-aws #23 @ialidzhikov
Requires new RBAC permissions - list and watch access for PodDisruptionBudgets in the target cluster.
```

### CODE REVIEW

**Review by prashanth26 — APPROVED** [review]
/lgtm 
/test

However, merge this after merging this dependent PR - https://github.com/gardener/gardener-extension-provider-aws/pull/324

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**
